esinstall/src/index.ts

```diff
@@ -3,7 +3,12 @@ import rollupPluginCommonjs, {RollupCommonJSOptions} from '@rollup/plugin-common
 import rollupPluginJson from '@rollup/plugin-json';
 import rollupPluginNodeResolve from '@rollup/plugin-node-resolve';
 import {init as initESModuleLexer} from 'es-module-lexer';
+import {build as esbuild, Metadata} from 'esbuild';
+import {invert, merge} from 'lodash/fp';
+import {mapKeys} from 'lodash';
 import fs from 'fs';
+import toUnixPath from 'slash';
```

Comment:
After doing this manually in so many different ways in Snowpack, I don’t mind deferring this to a package. We should make more use of this in general.